test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -253,7 +253,8 @@ def _run_dependency_check_test(image, ec2_connection, processor):
         "mxnet": ["1.9"],
         "pytorch": [],
         "huggingface_pytorch": ["1.8"],
-        "huggingface_tensorflow": ["2.4"]
+        "huggingface_tensorflow": ["2.4"],
+        "autogluon": ["0.2", "0.3"],
```

Comment:
I'm not sure we can add the skips for this, since openssl is found in the base image. Are we upgrading openssl in the AG containers?

Reply:
Added explicit upgrades to all images.